fineract-provider/src/main/java/org/apache/fineract/portfolio/charge/api/ChargesApiResourceSwagger.java

```diff
@@ -188,7 +188,7 @@ static final class GetChargesTemplateLoanChargeCalculationTypeOptions {
             private GetChargesTemplateLoanChargeCalculationTypeOptions() {}
 
             @Schema(example = "1")
-            public Long id;
+            public Integer id;
```

Comment:
This looks wrong to me...It should be Long based on the EnumOptionData line:36

Reply:
Done.